cfgov/unprocessed/js/modules/util/data-hook.js

```diff
@@ -60,8 +60,22 @@ function contains( element, value ) {
   return values.indexOf( value ) > -1 ? true : false;
 }
 
+/**
+ * @param {string} behavior
+ *  Behavior type used to find the element within the dom.
+ * @returns {HTMLNode} if it exists in the dom, null otherwise.
+ */
+function get( behavior ) {
```

Comment:
What about `find`? That might better fit with the other single term functions in this file.